Cubical/Categories/Instances/SievesAsFunctors.agda

```diff
@@ -0,0 +1,37 @@
+{-# OPTIONS --safe #-}
+
+module Cubical.Categories.Instances.SievesAsFunctors where
+
+-- There are also sieves in Cubical.Site.Sieve
```

Comment:
I think it would be good to also have a pointer there.
Maybe it would also be good to mention that the definitions are equivalent.